sqlglot/parser.py

```diff
@@ -638,6 +636,10 @@ def _parse_create(self):
         replace = self._match(TokenType.OR) and self._match(TokenType.REPLACE)
         temporary = self._match(TokenType.TEMPORARY)
         unique = self._match(TokenType.UNIQUE)
+        materialized = self._match(TokenType.MATERIALIZED)
+
+        if self._match_pair(TokenType.TABLE, TokenType.FUNCTION, advance=False):
+            self._match(TokenType.TABLE)
```

Comment:
is there no difference between create table function and create function? you seem to parse both the same way?

Reply:
This is to support BigQuery, see https://github.com/tobymao/sqlglot/pull/508/files#diff-b032342ced080bbc36b7733c3c2d2b87844cd21198ad5b15706668ba3c00aec3R259